src/main/resources/templates/registerNewMaterial.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <title>RegisterNewMaterial</title>
+    <meta charset="UTF-8">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="../public/css/styles.css" th:href="@{/css/styles.css}" rel="stylesheet"/>
+</head>
+
+<body>
+
+<div class="main-container">
+    <div th:replace="~{fragments :: header('registrerVare', ~{::a.nav-bar-button})}">
+        <a class="nav-bar-button" href="/" th:href="@{/}">Tilbage</a>
+    </div>
```

Comment:
This needs to be updated to match the new fragment requirements. Look at login.html for reference. Pay attention to details